transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/src/main/resources/js/typeMappingsSanitizer.js

```diff
@@ -134,19 +134,17 @@ function includesTypeNames(inputMap) {
     if (flagMatch) {
         return JSON.parse(flagMatch[1]);
     }
-    var majorVersion = inputMap.properties.version.major;
-    if (majorVersion) {
-        if (majorVersion >= 7) {
-            return false;
-        } else if (majorVersion <= 6) {
-            return true;
-        } else {
-            throw new Error("include_type_name was not set on the incoming URI." +
-                "The template needs to know what version the original request was targeted for " +
-                "in order to properly understand the semantics and what was intended.  " +
-                "Without that, this transformation cannot map the request " +
-                "to an unambiguous request for the target");
-        }
+    var majorVersion = inputMap.properties?.version?.major;
```

Comment:
It would be nice to flatten the inputMap into parameters.  This was vestigial from when it wasn't always possible or easy to send multiple values in a jinja template.